lib/testers/rule-tester.js

```diff
@@ -490,7 +490,19 @@ RuleTester.prototype = {
                 }
 
                 if (item.hasOwnProperty("output")) {
-                    const fixResult = SourceCodeFixer.applyFixes(eslint.getSourceCode(), messages);
+                    let fixResult = SourceCodeFixer.applyFixes(eslint.getSourceCode(), messages);
+
+                    if (item.multipass) {
+                        for (let i = 0; i < 10; i++) {
```

Comment:
I feel like it's not ideal to use 10 as a magic number here since the constant could change in [CLIEngine](https://github.com/eslint/eslint/blob/6445d2a55af7e3b797c5f7a187227cf267e0017c/lib/cli-engine.js#L137).